orchestration/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.3.7
```

Comment:
Is this on purpose?

Reply:
I did not explicitly change this - should I back this out?

Reply:
Yeah, please do.

Reply:
leaving this at 1.3.7 as per our discussion